pom.xml

```diff
@@ -23,7 +23,8 @@
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 
         <!-- dependencies -->
-        <junit.version>5.5.1</junit.version>
+        <junit.jupiter.version>5.5.1</junit.jupiter.version>
+        <junit.version>4.12</junit.version>
```

Comment:
I'd like to avoid having two versions of JUnit. Is 4 in mandatory to `TopologyTestDriver`?